nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -29,72 +29,100 @@
 - All [ROCm](https://rocm.docs.amd.com/en/latest/) packages have been updated to 5.7.0.
   - [ROCm](https://rocm.docs.amd.com/en/latest/) package attribute sets are versioned: `rocmPackages` -> `rocmPackages_5`.
 
-## New Services {#sec-release-23.11-new-services}
```

Comment:
I think keeping services is better because modules doesn't say a lot for non-experts I imagine

Reply:
I think it is also confusing for non-experts to talk about new services and having in the listing things like `networking.*`, `programs.*` and `hardware.*`. Not all are services/daemons/servers.

What would make most sense then:
  - Use `New configuration modules` (after all, NixOS is a declarative OS, so non-experts should also be exposed to modules as OS configuration).
  - Separate them all into `What's new`/[`Services`, `Programs`, etc] sections.

Reply:
@RaitoBezarius how does it look to you now?